viewer/packages/rendering/src/glsl/sector/primitives/eccentricCone.frag

```diff
@@ -13,12 +12,15 @@
 #define PI_HALF 1.5707963267949
 
 uniform sampler2D colorDataTexture;
-uniform sampler2D overrideVisibilityPerTreeIndex;
 uniform sampler2D matCapTexture;
 
 uniform vec2 treeIndexTextureSize;
 
 uniform mat4 projectionMatrix;
+uniform int renderMode;
+
+//Fix for iOS primitives not getting depth value if included before all uniform are declared
```

Comment:
```suggestion
// Note! Must be placed after all uniforms in order for this to work on iOS (REV-xyz)
```
Comment doesn't say anything about what can be done to this line and what cannot be done. Also add/reference a Jira issue with more details on the issue.